src/main/scala/com/fulcrumgenomics/umi/Umis.scala

```diff
@@ -80,8 +80,9 @@ object Umis {
       Some(name.substring(idx + 1, name.length))
     }
 
-    val umi   = rawUmi.map(raw => (if (raw.indexOf('+') > 0) raw.replace('+', '-') else raw).toUpperCase)
-    val valid = umi.forall(u => u.forall(isValidUmiCharacter))
+    val umiSeq = rawUmi.map(seq => (if (prefix.isEmpty) seq else seq.stripPrefix(prefix.get)))
+    val umi = umiSeq.map(raw => (if (raw.indexOf('+') > 0) raw.replace('+', '-') else raw).toUpperCase)
+    val valid  = umi.forall(u => u.forall(isValidUmiCharacter))
```

Comment:
1. Align the equals on the second assignment to match the code base.  We call this tim-format
2. Avoid the use of `get` on options (this is considered bad form), instead use `match`
```suggestion
    val umiSeq = prefix match {
      case None.     => rawUmi
      case Some(pre) => rawUmi.map(_.stripPrefix(pre))
    }
    val umi    =  umiSeq.map(raw => (if (raw.indexOf('+') > 0) raw.replace('+', '-') else raw).toUpperCase)
    val valid  = umi.forall(u => u.forall(isValidUmiCharacter))
```

Reply:
> Align the equals on the second assignment to match the code base. We call this tim-format

@nh13 out of curiosity, what tools support this format for linting/formatting/etc?

Reply:
Not sure, but something folks have tried previously. @clintval ?

Reply:
Found it! https://scalameta.org/scalafmt/

Reply:
I use this plugin while selecting the lines I want aligned.

https://plugins.jetbrains.com/plugin/13903-smart-align

It works most of the time pretty well.